products/jbrowse-cli/src/base.ts

```diff
@@ -49,67 +49,29 @@ export default abstract class JBrowseCommand extends Command {
   }
 
   async readJsonConfig(location: string) {
-    let locationUrl: URL | undefined
-    try {
-      locationUrl = new URL(location)
-    } catch (error) {
-      // ignore
-    }
-    if (locationUrl) {
-      const response = await fetch(locationUrl)
-      if (response.ok) {
-        return response.json()
-      }
-      throw new Error(`${response.statusText}`)
-    }
     return fsPromises.readFile(location, { encoding: 'utf8' })
   }
 
   async resolveFileLocation(location: string, check = true, warning = false) {
     let locationUrl: URL | undefined
-    let locationPath: string | undefined
     try {
       locationUrl = new URL(location)
     } catch (error) {
       // ignore
     }
     if (locationUrl) {
-      let response
-      try {
-        if (check) {
-          response = await fetch(locationUrl, { method: 'HEAD' })
-          if (response.ok) {
-            return locationUrl.href
-          }
+      if (check) {
+        const response = await fetch(locationUrl, { method: 'HEAD' })
+        if (!response.ok) {
           throw new Error(`${response.statusText}`)
```

Comment:
Could probably use a suggestion here so that users know which file gave a bad response.